uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -908,7 +908,7 @@ public function upgrade_1037() {
     // Create the menu item only if it doesn't exist:
     if ($activityTypeMenuItem['count'] === 0) {
       civicrm_api3('Navigation', 'create', [
-        'label' => 'Activity Types',
+        'label' => 'Task and Document Types',
         'name' => 'activity_types_administer',
```

Comment:
Shouldn't we change the name too to reflect the label. Since the former name reflects the label.